doc/cabal-commands.rst

```diff
@@ -1413,7 +1413,10 @@ A list of all warnings with their constructor:
 - ``missing-conf-script``: missing ``configure`` script with ``build-type: Configure``.
 - ``unknown-directory``: paths refer to a directory which does not exist.
 - ``no-repository``: missing ``source-repository`` section.
-- ``no-docs``: missing expected documentation files (changelog).
+- ``no-docs``: missing expected documentation files (changelog). Checks
```

Comment:
Feels like "changelog" appear too much now. I think the first occurrence (in the parenthesis) can be avoided.
```suggestion
- ``no-docs``: missing expected documentation files. Checks
```

Reply:
I let that pass because it was there to begin with.